pkg/server/uuid.go

```diff
@@ -48,11 +48,16 @@ func NewUUIDContext(id xid.ID) uuidContext {
 	return uuidContext{id}
 }
 
-// getOrSetUUID either reads the identifier file or creates it if it doesn't exist.
+// getOrSetUUID either reads the identifier file or creates it if it doesn't exist
+// unless useExternalIdentifier is set to true, in which case the local identifier file is ignored.
```

Comment:
I think the local identifier has higher precedence. And if present, it should match the one fetched from remote storage.
If the local identifier is not present, it then could use the remote one if instructed.

Reply:
makes perfect sense, now we use the local id if it's found even if the flag is on.